### ISSUE TITLE
Is there any way to control account in contract?

### ISSUE THREAD

**Issue by ksmin295** [qtumproject/qtum#550]
Hello, I'm studying how to make a smart contract in Qtum blockchain.

I can only find basical developer guide and simple example, but cannot find documentation for developing in detail.
Is there any source I can get help, plz tell me.

And I wanna control account in smart contract. (e.g creating new account, reset password of account)
If there is no method to control it, I'm thinking to deal with it through external server using Qtumjs and rpc like the Oraclize in Ethereum.

In short, there are 3 questions.
1. Detailed documentation for Qtum dApp developer
2. Any method to controlling account in contract?
3. Is it possible to send HTTP POST request to external server? (like Oracleize)

Always thank you to Qtum team's effort.

**Comment by Baiqiang** [comment]
1. Please have a look at Qtum Book: https://book.qtum.org/
2. I don't think we have "account".
3. No.

**Comment by ksmin295** [comment]
Thanks.
